pom.xml

```diff
@@ -120,10 +120,10 @@
 
         <!-- Version Properties -->
         <opendj.server.version>2.6.4</opendj.server.version>
-        <opendj.sdk.version>2.6.9</opendj.sdk.version>
+        <opendj.sdk.version>2.6.11</opendj.sdk.version>
```

Comment:
2.6.11 should be better than 2.6.9, but it feels like this should be tackled in a different PR